src/assets/translations/cs-CS.json

```
@@ -2151,6 +2151,7 @@
       "External ID not found": "Externí ID nenalezeno",
       "External ID verified successfully": "Externí ID úspěšně ověřeno",
       "External Id": "Externí id",
+      "Reversal External Id": "Externí ID storna",
```

Biome (2.5.6)

[error] The key Reversal External Id was already declared.

(lint/suspicious/noDuplicateObjectKeys)